sharktank/sharktank/kernels/batch_matmul_transpose_b.py

```diff
@@ -80,7 +80,7 @@ def generate(self, ksel: KernelSelection, kb: KernelBuilder):
         spec_sig = f"L{a_ident}_R{b_ident}"
         template_file = "batch_matmul_transpose_b.mlir"
         target_function_name = f"sharktank_batch_matmul_transpose_b_{spec_sig}"
-
+        cst_zero = "0." if "f" in str(accum_type) else "0"
```

Comment:
Be specific. Don't just use stringy char-in-str.

In this case, I think you can say something like:

```
cst_zero = "0" if IntegerType.isa(accum_type) else "0."
```